README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
```

Comment:
Well... no, you said above what the mission is, "support the community" is not in that mission. The use of "community" throughout this document may be unhelpful given how difficult it is to define the term, it's nice to use when you handwave but this ought not be a handwavy document, we should attempt to find more concrete terms if we are going this direction.

"The size and scope of the work governed by the TSC will grow as is required in order to responsibly meet the goals of the Node.js Foundation."